.idems_app/deployments/plh/global.config.ts

```diff
@@ -3,26 +3,36 @@ import { DEFAULT_CONSTANTS } from "data-models";
 
 const app_constants = DEFAULT_CONSTANTS;
 /** Define default startup actions for plh app */
-app_constants.APP_INITIALISATION_DEFAULTS.app_first_launch_actions = [
+app_constants.APP_INITIALISATION_DEFAULTS.app_launch_actions = [
   {
     type: "template_popup",
     value: "splash_screens",
+    first_launch_only: true,
+    condition: null,
   },
   {
     type: "template_popup",
     value: "accept_terms",
+    first_launch_only: false,
+    condition: "terms_not_accepted"
```

Comment:
Ideally we should be tracking via a localstorage variable, e.g. `@fields.terms_v1_accepted`, populating that field on template condition. Ideally this can be handled all within authoring if we move this code to templating syntax instead of hardcoded